skiko/build.gradle.kts

```diff
@@ -154,6 +156,33 @@ kotlin {
         withJava()
     }
 
+    val nativeTarget = when (target) {
+        "macos-x64" -> macosX64()
+        // "linux-x64" -> linuxX64()
+        // "windows-x64" -> mingwX64()
+        else -> null
+    }
+    nativeTarget?.apply {
+        compilations.getByName("main") {
+            val skia by cinterops.creating {
+                defFile("src/nativeInterop/cinterop/skia.def")
+
+                val skiaDir = skiaDir.get().absolutePath
+                val kotlinNativeDataPath = System.getenv("KONAN_DATA_DIR")?.let { File(it) }
+                    ?: File(System.getProperty("user.home")).resolve(".konan")
+
+                compilerOpts(
+                     // TODO: Why do we need `include/c++/v1`?
```

Comment:
Do we really need it C++ headers now?

Reply:
Yes for now. There will be a corresponding fix to native that fixes that.